src/main/resources/add-ons-to-index.json

```diff
@@ -4220,6 +4261,7 @@
       "tags": [
         "reporting"
       ],
+      "icon": "bar-chart",
```

Comment:
Irrelevant as this module not exactly modifying reports. Kindly check its usage and delete/modify this accordingly